pkg/cluster/k8sres.go

```diff
@@ -1462,10 +1481,13 @@ func (c *Cluster) generateLogicalBackupJob() (*batchv1beta1.CronJob, error) {
 			},
 		}}
 
+	annotations := c.generatePodAnnotations(nil)
```

Comment:
This would cause LogicalBackupJobs to only get annotations from the Operator config, not the Cluster spec.
I think you can change it to c.generatePodAnnotations(c.Spec) to make it pick up annotations from the Cluster spec.

Reply:
Good point, thanks!